File: Java/1837.java
import java.util.Scanner;

public class 1837 {
    public static void main(String[] args) {
        Scanner read = new Scanner(System.in);
        int a, b, q, r;
        a = read.nextInt();
        b = read.nextInt();
        r = (a % Math.abs(b));
        if (r >= 0) {
            q = (a - r)/b;
            System.out.printf("%d %d\n", q, r);
        }
        else {
            r += Math.abs(b);
            q = (a - r)/b;
            System.out.printf("%d %d\n", q, r);
        }
    }
}

File: C++/1837.cpp
#include <iostream>
using  namespace std;

int main(){
    int a, b, q, r;
    cin >> a >> b;

    r = (a % abs(b));

    if (r >= 0){
        q = (a - r)/(b);
        cout << q << " " << r << endl;
    }
    else {
        r += abs(b);
        q = (a - r)/(b);
        cout << q << " " << r << endl;
    }

    return 0;
}
